### PR TITLE
docs: update HOWTO.md

### PR DESCRIPTION
**Pull request by rdotjain**
add instructions to setup config environment variables when running in a dev container

### CODE REVIEW

**Review by nullr0ute — APPROVED** [review]
LGTM

File: HOWTO.md [hunk 1]
```diff
@@ -533,8 +533,11 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    `/usr/share/fdo`, `/etc/fdo/` or
    `/etc/fdo/manufacturing-server.conf.d/`. The paths will be checked in that
    same order.
+   
+   Additionally, set up the `MANUFACTURING_SERVER_CONF` environment variable to point
+   to the path of the configuration file if running in a dev container.
 
-3. Execute `fdo-manufacturing-server` or run it as a service, see sample
+4. Execute `fdo-manufacturing-server` or run it as a service, see sample
```
Comment (7flying) [inline HOWTO.md:540]: This should have been step 3 still

File: HOWTO.md [hunk 2]
```diff
@@ -548,8 +551,11 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    either in `/usr/share/fdo`, `/etc/fdo/` or
    `/etc/fdo/owner-onboarding-server.conf.d/`. The paths will be checked in
    that same order.
+   
+   Additionally, set up the `OWNER_ONBOARDING_SERVER_CONF` environment variable to point
+   to the path of the configuration file if running in a dev container.
 
-3. Generate an Ownership Voucher (see [How to generate an Ownership
+4. Generate an Ownership Voucher (see [How to generate an Ownership
```
Comment (7flying) [inline HOWTO.md:558]: same in here

File: HOWTO.md [hunk 3]
```diff
@@ -580,8 +589,11 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    Files/serviceinfo-api-server.yml](#serviceinfo-api-serveryml), and place it either in
    `/usr/share/fdo`, `/etc/fdo/` or `/etc/fdo/serviceinfo-api-server.conf.d/`. The
    paths will be checked in that same order.
+      
+   Additionally, set up the `SERVICEINFO_API_SERVER_CONF` environment variable to point
+   to the path of the configuration file if running in a dev container.
 
-2. Execute `fdo-serviceinfo-api-server` or run it as a service, see sample file
+3. Execute `fdo-serviceinfo-api-server` or run it as a service, see sample file
```
Comment (7flying) [inline HOWTO.md:596]: 2

@rdotjain , can you fix this in another PR? thanks

Comment (rdotjain) [reply HOWTO.md:596]: thanks for the review! but this seems like an issue with gh diff, because if we view the actual file the bullet points remain intact

Comment (7flying) [reply HOWTO.md:596]: It doesn't look like an issue with gh diff. When the file is rendered in html, the markdown to html transpiler is smart enough to correctly number the lists, but if you see the file in raw format (markdown) you'll see that the numbers are not correct.

Comment (7flying) [reply HOWTO.md:596]: @rdotjain please take a look

Comment (rdotjain) [reply HOWTO.md:596]: Thanks for pointing it out! Fixed in #576

**Pull request merged**